OpenQA/Isotovideo/Utils.pm

```diff
@@ -321,6 +321,10 @@ sub handle_generated_assets ($command_handler, $clean_shutdown) {
             my $dir = 'assets_private';
             my $name = $bmwqemu::vars{"STORE_HDD_$i"} || undef;
             unless ($name) {
+                if ($bmwqemu::vars{"PUBLISH_HDD_$i"} =~ /none/i) {
+                    bmwqemu::log_call("Asset upload is skipped for PUBLISH_HDD_$i");
+                    next;
+                }
```

Comment:
This can probably be moved below the next line, where we already assign `$name`.
```suggestion
                if ($name =~ /none/i) {
                    bmwqemu::log_call("Asset upload is skipped for PUBLISH_HDD_$i=$name");
                    next;
                }
```